README.md

```diff
@@ -32,10 +30,26 @@ jobs:
 
 ### Inputs
 
-| **Name**  | **Required** | **Description**                                                                                               | **Type** |
-|-----------|--------------|---------------------------------------------------------------------------------------------------------------|----------|
-| `version` | Yes          | Version to install, e.g. `nightly` or `1.0.0`.  **Note:** Foundry only has nightly builds for the time being. | string   |
+| **Name**  | **Required** | **Default** | **Description**                                                                                              | **Type** |
+| --------- | ------------ | ----------- | ------------------------------------------------------------------------------------------------------------ | -------- |
+| `version` | No           | `nightly`   | Version to install, e.g. `nightly` or `1.0.0`. **Note:** Foundry only has nightly builds for the time being. | string   |
+
+### RPC Caching
+
+This action matches Forge's behavior and caches all RPC responses in the `~/.foundry/cache/rpc` directory. This is done to
+speed up the tests and avoid hitting the rate limit of your RPC provider.
+
+The logic of the caching is as follows:
+
+- Always load the latest valid cache, and always create a new one with the updated cache.
+- When there are no changes to the fork tests, the cache does not change but the restore key does, since the key is based on the commit hash.
+- When the fork tests are changed, both the cache and the store key are updated.
```

Comment:
Got it, I see. And is "store key" the same as the "restore keys" in the actions/cache?

Reply:
Oh my goodness, I have completely butchered the terminology in the proposed README. I'm sorry, I must have been off-caffeine on the day when I opened the PR.

I have pushed a commit to fix the wording, you can see the diff here:

https://github.com/foundry-rs/foundry-toolchain/pull/19/commits/607b0c57e2b983f674df54e67152ee9ccb785cfa

Basically, I should have just said "key", because the "restore keys" never change on the same OS platform. It's the "key" (simple key, no restore) that changes all the time because it's dependent upon the commit hash.